spec/beaker/test_suite_spec.rb

```diff
@@ -24,6 +24,27 @@ module Beaker
         expect(tfs).to include rb_test
       end
 
+      it 'defaults to :slow fail_mode is not provided through parameter or options' do
+        @files = [ rb_test ]
+        ts = Beaker::TestSuite.new('name', 'hosts', options, Time.now)
+        tfm = ts.instance_variable_get(:@fail_mode)
```

Comment:
This is one of those cases where I prefer to expose an attribute for the options the instance was configured with, and test via that route instead of via `instance_variable_get`, but definitely don't consider this a blocker, or even a change request, really.